$MirOS: ports/www/lynx/patches/patch-samples_lynx_lss,v 1.3 2008/03/09 18:19:43 tg Exp $

	usability

--- samples/lynx.lss.orig	Tue Sep 19 00:28:05 2006
+++ samples/lynx.lss	Wed Dec 17 00:14:05 2008
@@ -1,3 +1,5 @@
+# $MirOS: src/gnu/usr.bin/lynx/samples/lynx.lss,v 1.5 2008/10/14 00:40:50 tg Exp $
+#
 # Setting the normal and default types lets us keep (almost) the same colors
 # whether the terminal's default colors are white-on-black or black-on-white.
 # It is not exact since the default "white" is not necessarily the same color
@@ -6,7 +8,7 @@
 # If you really want the terminal's default colors, and if lynx is built using
 # ncurses' default-color support, remove these two lines:
 normal:		normal:			lightgray:black
-default:	normal:			white:black
+default:	normal:			lightgray:black
 
 # Normal type styles correspond to HTML tags.
 #
@@ -113,3 +115,12 @@ span.htmlsrc_badseq:normal:		red
 span.htmlsrc_badtag:normal:		red
 span.htmlsrc_badattr:normal:		red
 span.htmlsrc_sgmlspecial:normal:	yellow
+
+# Workarounds for specific sites
+div.normaltext:		normal
+div.storytext:		normal
+span.user_bold:		bold:		red
+span.user_italic:	bold:		brightblue
+span.user_underlined:	underline:	brown
+div.user_quote:		normal:		white
+div.user_quote_name:	normal:		yellow
